src/index.html

```diff
@@ -10,7 +10,40 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+  
+    <form action="#"
+      method="POST"
+      class="form form--big"
+      data-qa="big"
+    >
```

Comment:
Need to fix everywhere
```suggestion
    <form
      action="#"
      method="POST"
      class="form form--big"
      data-qa="big"
    >
```